components/MainLayout.js

```diff
@@ -15,6 +16,10 @@ export default function MainLayout ({ children }) {
     <div>
       <Head>
         <title>React Bangkok 2.0.0</title>
+        <link rel='apple-touch-icon' href={reactbkk} />
+        <link rel='apple-touch-startup-image' href={reactbkk} />
+        <meta name='apple-mobile-web-app-title' content='React Bangkok 2.0' />
```

Comment:
I think this can be shortened, because there is not enough space for the icon text.